STANDARDS.md

```diff
@@ -19,54 +69,60 @@ Each dataset shall be versioned.
 
-## Force Field Releases
+Each version increment should take the information from the previous `changelog` field, and add a new entry of the form { "version": description } that explains the modifications made to the dataset. Each dataset should therefore have the complete changelog.
 
-When a new force field is released, a dataset corresponding to all results used for the force field fitting should be created.
-This gives a single reference for these data instead of many references.
-The format of these dataset names is:
+# Fitting standards
 
-    `"OpenFF Force Field <friendly name> <ff version>"`
+Reference level of theory: `B3LYP/DZVP`
+Geometry optimizations: `geomeTRIC` using the TRIC coordinate system
+QM program: `Psi4`
 
-## Group
+For unconstrained geometries, all molecules must have:
+- Wiberg Bond Orders (parameter interpolation)
+- Hessian (frequency fitting) (is this too restrictive? discuss)
 
-The dataset's group should be set to `"OpenFF"`.
+Pre-submission filtering:
+- Unless explicitly specified in the submission descriptions, torsion drives must be on 4 connected atoms
+- Torsions driving a ring will give a warning, and torsions in a a ring of  3,4,5,6 atoms is considered an error
+- Warnings will be given if an atom does not have a complete valence set
 
-## Molecule validation
+Post-submission filtering: (we need testers/validators for these)
+	- OpenFF toolkit ingestion with strict stereo checking in RDKit
+	- Hydrogen bonding
+	- Torsion drives on rings or other high barrier issues
+	- CMILES change (no way to build new CMILES since connectivity cannot be trusted, also this means we can't trust per-molecule CMILES until this filtering check passes)
 
-* See ["Molecule submission checklist"](https://github.com/openforcefield/qcsubmit/issues/9)
+## Force Field Releases
 
-## Standard functions and modules for entry preparation
+Upon fitting for a new force field release, for the purpose of paper publication, public reference, etc, all molecules should be placed in a single dataset (per type). This gives a single reference for these data instead of many references. Filtering must be done prior, such that all molecules in the release datasets pass all post-submission filters.
 
-* QCSubmit (https://github.com/openforcefield/qcsubmit)
+The format of these dataset names must use the following format:
 
-## Related/ongoing discussions
+    `"OpenFF SMIRNOFF <friendly name> <ff version>"`
 
-### Required fields
+for example, all datasets (optimizations, torsion drives, and Hessians) with the name `"OpenFF SMIRNOFF Sage 2.0.0"` would refer to all data used to train `openff-2.0.0.offxml` in the `openforcefields` package.
 
-* See ["Fields that should be required for OpenFF submissions"](https://github.com/openforcefield/qcsubmit/issues/3)
+Besides the regular information from the other datasets, these fitting datasets must have:
 
+- `DOI`
+- Dataset standards version (this document)
```

Comment:
Awesome. This is some excellent provenance thinking! Great work Trevor!

Reply:
I think having the compliance status with an explicit `V3` status will also help with this nicely.